ci/test/00_setup_env_mac.sh

```diff
@@ -16,3 +16,4 @@ export RUN_FUNCTIONAL_TESTS=false
 export GOAL=""
 export DEP_OPTS="NO_QT=1"
 export GRIDCOIN_CONFIG="--enable-reduce-exports"
+export NEED_XVFB="false"
```

Comment:
Seems we didn't need Xvfb for Travis?

Reply:
Cross mac doesn't run the tests, so no need for xvfb.